scripts/install

```diff
@@ -102,35 +110,24 @@ ynh_setup_source --dest_dir="$final_path"
 mkdir -p "$final_path/plugins/discourse-ldap-auth"
 ynh_setup_source --dest_dir="$final_path/plugins/discourse-ldap-auth" --source_id=ldap-auth
 
-#=================================================
-# INSTALL RUBY
-#=================================================
-ynh_script_progression --message="Installing Ruby..."
-
-ynh_install_ruby --ruby_version=$RUBY_VERSION
+chmod -R 750 "$final_path"
+chmod -R o-rwx "$final_path"
+chown -R $app:www-data "$final_path"
```

Comment:
```suggestion
chmod 750 "$final_path"
chown -R $app:www-data "$final_path"
```

It's a bit odd to set 750 recursively because that'll also affect regular files, and for most of them the `x` bit don't make sense as they ain't executable

The most important thing with permission is that the initial "parent" folder should be protected, which corresponds to the `o-rwx` (or `xx0`, whichever)